core/templates/dev/head/pages/exploration_editor/EditorServices.js

```diff
@@ -843,14 +846,16 @@ oppia.factory('explorationStatesService', [
     var saveStateProperty = function(stateName, backendName, newValue) {
       var oldValue = getStatePropertyMemento(stateName, backendName);
       var newBackendValue = angular.copy(newValue);
+      var oldBackendValue = angular.copy(oldValue);
 
       if (BACKEND_CONVERSIONS.hasOwnProperty(backendName)) {
         newBackendValue = convertToBackendRepresentation(newValue, backendName);
+        oldBackendValue = convertToBackendRepresentation(oldValue, backendName);
       }
 
       if (!angular.equals(oldValue, newValue)) {
         changeListService.editStateProperty(
-          stateName, backendName, newBackendValue, oldValue);
+          stateName, backendName, newBackendValue, oldBackendValue);
```

Comment:
It looks like there's no error because there's no validation that occurs on the backend for old value. In fact it doesn't seem to ever be accessed on the backend. It's defined as a property but never used. Even if I remove this now, there's no error. I just noticed it and thought it should be changed.